web/pandas/pdeps/0010-required-pyarrow-dependency.md

```diff
@@ -1,12 +1,48 @@
 # PDEP-10: PyArrow as a required dependency for default string inference implementation
 
-- Created: 17 April 2023
-- Status: Accepted
+- Created: 17 April 2023 (updated May 8, 2024)
+- Status: Rejected
 - Discussion: [#52711](https://github.com/pandas-dev/pandas/pull/52711)
               [#52509](https://github.com/pandas-dev/pandas/issues/52509)
 - Author: [Matthew Roeschke](https://github.com/mroeschke)
           [Patrick Hoefler](https://github.com/phofl)
-- Revision: 1
+- Revision: 2
+
+# Note
+
+This PDEP was originally accepted on May 8, 2023. However, after reviewing feedback posted
+on the feedback issue [#54466](https://github.com/pandas-dev/pandas/issues/54466), we, the members of
+the core team, have not decided with moving forward with this PDEP for pandas 3.0.
+
+The primary reasons for rejecting this PDEP are twofold:
+
+1) Requiring pyarrow as a dependency causes installation problems.
+   - Pyarrow does not fit or has a hard time fitting in space-constrained environments 
```

Comment:
I think what we could learn from this process is what caused this to change our minds? These issues were discussed leading up to the acceptance of PDEP-10. 

The way this is written I think reads more as "we discovered this after the fact" instead of "we decided that X amount of negative feedback on these points was enough to revert". I think there is some value to future PDEPs to set expectations around the latter